src/core/ext/transport/chttp2/transport/chttp2_transport.cc

```diff
@@ -214,6 +215,9 @@ grpc_core::CallTracerInterface* CallTracerIfEnabled(grpc_chttp2_stream* s) {
           s->context)[GRPC_CONTEXT_CALL_TRACER_ANNOTATION_INTERFACE]
           .value);
 }
+
+grpc_core::WriteTimestampsCallback write_timestamps_callback_g = nullptr;
```

Comment:
`g_write_timestamps_callback` - we always prefix the g for global, never suffix it

Reply:
(same below too)

Reply:
Fixed.